### PR TITLE
Enable Zip Testing in Sandbox

### PR DESCRIPTION
**Pull request by Trenly**
null

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/68822)

### CODE REVIEW

File: Tools/SandboxTest.ps1 [hunk 1]
```diff
@@ -336,4 +336,4 @@ $Script
 
 Write-Host
 
-WindowsSandbox $SandboxTestWsbFile
\ No newline at end of file
+WindowsSandbox $SandboxTestWsbFile
```
Comment (OfficialEsco) [inline Tools/SandboxTest.ps1:339]: Why are you this way GitHub
```suggestion
WindowsSandbox $SandboxTestWsbFile


```

**Review by OfficialEsco — DISMISSED** [review]

File: Tools/SandboxTest.ps1 [hunk 2]
```diff
@@ -171,7 +171,7 @@ $bootstrapSettingsContent['logging'] = @{level = 'verbose' }
 if ($EnableExperimentalFeatures) {
   $bootstrapSettingsContent['experimentalFeatures'] = @{
     dependencies    = $true
-    portableInstall = $true
```
Comment (lechacon) [inline Tools/SandboxTest.ps1:174]: Please update the comment above

File: Tools/SandboxTest.ps1 [hunk 3]
```diff
@@ -171,7 +171,7 @@ $bootstrapSettingsContent['logging'] = @{level = 'verbose' }
 if ($EnableExperimentalFeatures) {
   $bootstrapSettingsContent['experimentalFeatures'] = @{
     dependencies    = $true
```
Comment (lechacon) [inline Tools/SandboxTest.ps1:173]: Nit: Remove the extra space before the `=`. Looks like it used to be aligned with the line below, but it changed now

**Review by florelis — DISMISSED** [review]

**Review by ItzLevvie — DISMISSED** [review]

**Review by OfficialEsco — APPROVED** [review]

**Review by zachcarp — APPROVED** [review]

**Comment by zachcarp** [comment]
@Trenly - Is this ready to merge?

**Comment by Trenly** [comment]
> @Trenly - Is this ready to merge?

I believe so

**Pull request merged**